start_session.php

```diff
@@ -126,6 +126,26 @@
             ]
         ],
     ],
+    'irmatube_premium' => [
+        'nextSession' => [
+            'url' => IRMATUBE_NEXT_SESSION_URL
+        ],
+        'request' => [
+            '@context' => 'https://irma.app/ld/request/disclosure/v2',
+            'disclose' => [
+                [
+                    ['pbdf.pbdf.facebook.fullname'],
+                    ['pbdf.gemeente.personalData.fullname'],
+                ]
+            ],
+        ]
+    ],
+    'watch_premium_contents' => [
+        '@context' => 'https://irma.app/ld/request/disclosure/v2',
+        'disclose' => [[[
+            'pbdf.IRMATube.member.fullname',
```

Comment:
This issuer does not exist, you [probably meant `pbdf.pbdf.irmatube.fullname`](https://privacybydesign.foundation/attribute-index/en/pbdf.pbdf.irmatube.html#pbdf.pbdf.irmatube.fullname).